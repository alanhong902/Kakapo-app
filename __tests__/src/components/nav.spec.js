import React from 'react';
import { Nav } from 'components/';
import { getData, createComponentWithIntl } from '../helper';

test('<Nav/> render', () => {
  const props = {
    ...getData('themes'),
  };
  const tree = createComponentWithIntl(<Nav {...props} />).toJSON();
  expect(tree).toMatchSnapshot();
});
