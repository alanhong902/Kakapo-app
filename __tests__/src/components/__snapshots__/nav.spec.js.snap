// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Nav/> render 1`] = `
<div
  className="topbar"
  style={
    Object {
      "backgroundColor": "#3b216a",
    }
  }
>
  <a
    className="download-app"
    href="http://www.kakapo.co/app.html"
    rel="noopener noreferrer"
    target="_blank"
  >
    <span>
      Download Kakapo for iOS / Android, Desktop & Chrome!
    </span>
  </a>
  <div
    className="share"
  >
    <div
      className="fb-share-button"
      data-href="http://www.kakapo.co"
      data-layout="button_count"
    />
    <a
      className="twitter-share-button"
      data-url="http://www.kakapo.co"
      href="https://twitter.com/share"
    >
      Tweet
    </a>
  </div>
</div>
`;
